packages/common-ui-web/src/components/DynamicSVG/hooks/types.ts

```diff
@@ -10,20 +10,20 @@ export interface DynamicSVGCustomizationFunctions {
    * groups this element is part of.
    *
    * @param element The element to apply the custom attributes to.
-   * @param groupIds The IDs of the SVG group elements this element is part of (the IDs are in order).
+   * @param groups SVG group elements this element is part of (the IDs are in order).
```

Comment:
Replace `the IDs are in order` with `the elements are in order` both here and in other similar TSDocs

Reply:
already replaced on all the other places.

Reply:
No, I mean replace the wording `(the IDs are in order)` with `(the elements are in order)` in the documentation sentences. Also do this in the other TSDocs in the file

Reply:
my bad